_populate_db/populate.py

```diff
@@ -1,20 +1,59 @@
 import os, sys, re
 import numpy as np
 from xml.etree import ElementTree
-import dicom
+try:
+    import dicom
+except ImportError:
+    import pydicom
 
 assert not os.path.exists(os.path.join(os.path.pardir, 'pylidc.sqlite')), "`pylidc.sqlite` already exists. Aborting."
 
+def getSettings():
```

Comment:
I don't see any particular reason to put this into a function. Let's just run this code block at the top. I like the idea of xml location settings in the config file, though.

Reply:
The idea for the function was to make it reusable in the future from other places in the library (`Scan` also needs it). It is also just a bit nice/more organized code that is easier to read.

Reply:
If this is the case, then it should maybe go in the `utils.py` module, from which it could be imported into `populate.py` or `Scan.py`. The import should not be in the function itself, and the function name should be `get_settings` to match pep 8 (and the coding style used throughout).

Reply:
Yes, indeed - I agree `utils` is a better place, but I think the code overall would need to reorganized, since right now `populate` is outside of the main library. I felt such reorganization is more than I can commit to.

I will rename as you suggest (sorry for not catching the convention), but I can't take on moving it to `utils` and re-organizing the code.